tests/transpilation/sha3.opcode

```diff
@@ -0,0 +1,3 @@
+PUSH1 0x20
+PUSH1 0x0
+SHA3
```

Comment:
Is it true that you compute keccak of 0 here?

Reply:
Yes